ui/apps/dashboard/src/components/Navigation/AppNavigation.tsx

```diff
@@ -32,8 +34,8 @@ type NavItem = {
 const ALL_ENVIRONMENTS_SLUG = 'all';
 const BRANCH_PARENT_SLUG = 'branch';
 
-export default async function AppNavigation({ environmentSlug }: AppNavigationProps) {
-  const isEventSearchEnabled = await getBooleanFlag('event-search');
+export default function AppNavigation({ environmentSlug }: AppNavigationProps) {
+  const { value: isEventSearchEnabled } = useBooleanFlag('event-search');
```

Comment:
What is the default here? I know not many accounts have this enabled, but I want to confirm that this doesn't show by default, does `useBooleanFlag` have an option for default value?

Reply:
It'll default to false unless the default is explicitly set to true:
https://github.com/inngest/inngest/blob/remove-rsc-on-envs-page/ui/apps/dashboard/src/components/FeatureFlags/hooks.ts#L14